### ISSUE TITLE
Defect BOM creation in v.1.4 

### ISSUE THREAD

**Issue by HSSE-Dev** [CycloneDX/cyclonedx-maven-plugin#15]
When creating a BOM from a Maven project, the URL is inserted twice in the BOM header under xsi:schemaLocation.
```
<?xml version="1.0" encoding="UTF-8"?>
<bom xmlns="http://cyclonedx.org/schema/bom/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" serialNumber="urn:uuid:82290a48-e0ea-4c33-9625-18e55f4ac348" version="1" xsi:schemaLocation="http://cyclonedx.org/schema/bom/1.1 http://cyclonedx.org/schema/bom/1.1">
    <components>
        <component type="library">
```

This behavior occurs with both commands _makeBom_ and _makeAggregateBom_
Can anyone confirm the behavior?

**Comment by pachulo** [comment]
I cannot reproduce this with version 1.4.1 of the maven plugin:

```
<?xml version="1.0" encoding="UTF-8"?>
<bom xmlns="http://cyclonedx.org/schema/bom/1.1" serialNumber="urn:uuid:6106f82d-a630-47b0-9e92-34e368f5a645" version="1">
    <components>

```

**Comment by stevespringett** [comment]
@pachulo is correct. This was resolved in v1.4.1 of the plugin. Specifically this change to CycloneDX Core Java 2.0.2 which is included in 1.4.1.

https://github.com/CycloneDX/cyclonedx-core-java/commit/22f8a6f2f84d3d55f3c1516ab09ace0bb2bab01d
